src/models/ssh-key.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) The Arvados Authors. All rights reserved.
+//
+// SPDX-License-Identifier: AGPL-3.0
+
+import { Resource } from '~/models/resource';
+
+export interface SshKey {
+    name: string;
```

Comment:
Duplicated interface - is it necessary? There's a SshKeyResource with the same keys.